### PR TITLE
Convert link hostnames in findYoutubeLinks

### PR DESCRIPTION
**Pull request by CDeLeon94**

### CODE REVIEW

**Comment by CDeLeon94** [comment]
Possible solution to #10 

Please review, untested, and I have low familiarity with TS.

**Comment by CDeLeon94** [comment]
Tried to create an account on feddit.rocks as a step towards having a testing, however, I'm not getting the email validation for some reason. 

But I was at least able to build the .ts to .js successfully

File: src/index.ts [hunk 1]
```diff
@@ -39,7 +39,14 @@ const findYoutubeLinks = (text: string): UrlLink[] => {
         }
     });
 
-    return youtubeLinks;
+    const pipedLinks = youtubeLinks.map(urlLinkObj => {
```
Comment (obosob) [inline src/index.ts:42]: I would suggest calling this "link" just as in the `filter` above, they're the same objects. makes it more readable.

**Comment by FireMasterK** [comment]
Thanks for the PR, but I decided to just use forEach and modify the link object since that is easier :)

**Pull request closed without merging**